samples/HttpServer_ConfigNetwork/include/AppSettings.h

```diff
@@ -24,34 +24,33 @@ struct ApplicationSettingsStorage {
 
 	void load()
 	{
-		DynamicJsonBuffer jsonBuffer;
+		DynamicJsonDocument doc(1024);
 		if(exist()) {
 			int size = fileGetSize(APP_SETTINGS_FILE);
 			char* jsonString = new char[size + 1];
 			fileGetContent(APP_SETTINGS_FILE, jsonString, size + 1);
-			JsonObject& root = jsonBuffer.parseObject(jsonString);
+			auto error = deserializeJson(doc, jsonString);
+			if(!error) {
+				JsonObject network = doc["network"].as<JsonObject>();
+				ssid = network["ssid"].as<String>();
+				password = network["password"].as<String>();
 
-			JsonObject& network = root["network"];
-			ssid = network["ssid"].asString();
-			password = network["password"].asString();
+				dhcp = network["dhcp"];
 
-			dhcp = network["dhcp"];
-
-			ip = network["ip"].asString();
-			netmask = network["netmask"].asString();
-			gateway = network["gateway"].asString();
+				ip = network["ip"].as<String>();
+				netmask = network["netmask"].as<String>();
+				gateway = network["gateway"].as<String>();
```

Comment:
`ssid`, `password`, `ip`, `netmask` and `gateway` should probably all use `as<const char*>()`